client/src/bundles/ConfirmFolder.js

```diff
@@ -60,7 +60,8 @@ jQuery.entwine('ss', ($) => {
   $('#Form_ConfirmFolderForm_FolderID_Holder .treedropdownfield.is-open,#Form_ItemEditForm_FolderID .treedropdownfield.is-open').entwine({
     onunmatch() {
       // Build url
-      const parsedURL = url.parse('admin/user-forms/getfoldergrouppermissions');
+      const adminUrl = window.location.pathname.split('/')[1];
```

Comment:
How safe it is to assume that admin controller is a single-level only and doesn't have any slashes in its path?

Reply:
Not. I will fix this by getting the admin_url from a data attribute.